src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle {
```

Comment:
- Write an interface `Figure` and 3 classes **implementing** it

you should add `implements Figure` to all of your classes